config_src/infra/FMS2/MOM_interp_infra.F90

```diff
@@ -4,20 +4,37 @@ module MOM_interp_infra
 ! This file is part of MOM6. See LICENSE.md for the license.
 
 use MOM_domain_infra,    only : MOM_domain_type, domain2d
+use MOM_io,              only : axis_info
+use MOM_io,              only : get_var_axes_info
 use MOM_time_manager,    only : time_type
-use horiz_interp_mod,    only : horiz_interp_new, horiz_interp, horiz_interp_init, horiz_interp_type
-use mpp_io_mod,          only : axistype, mpp_get_axis_data
-use time_interp_external_mod, only : time_interp_external
-use time_interp_external_mod, only : init_external_field, time_interp_external_init
-use time_interp_external_mod, only : get_external_field_size
-use time_interp_external_mod, only : get_external_field_axes, get_external_field_missing
+use MOM_error_handler, only : MOM_error, FATAL
+use MOM_string_functions, only : lowercase
+use horiz_interp_mod, only : horiz_interp_new, horiz_interp, horiz_interp_init, horiz_interp_type
```

Comment:
These module use statements for routines in MOM_io, MOM_error_handler and MOM_string_functions introduce a backward dependency on code in the src/framework directory and the potential for a predecessor cycle.  There are no other instances of config_src/infra code on src/framework code, and I think that we should keep it that way.  Please consider revising this dependency.

Reply:
These use statements have not created any predecessor cycles between modules.